lavalink/playermanager.py

```diff
@@ -99,7 +103,7 @@ def create(self, guild_id: int, region: str = 'eu', endpoint: str = None, node:
         node = self._lavalink.node_manager.find_ideal_node(region)
 
         if not node:
-            raise Exception('No available nodes!')  # TODO: NodeException or something
+            raise NodesNotFound('No available nodes!')
```

Comment:
rename to NodeException to make it more mainstream, no need to create 10 different exceptions.